install/docker/centos-docker-install.md

```diff
@@ -0,0 +1,44 @@
+# Installing Docker for Kata Containers on CentOS
+
+> **Note:**
+>
+> - This guide assumes you have
+>   [already installed the Kata Containers packages](../centos-installation-guide.md).
+
+1. Install the latest version of Docker with the following commands:
+
+   > **Note:** This step is only required in case Docker is not installed on the system.
+
+   ```bash
+   $ sudo yum-config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo
+   $ sudo yum -y install docker-ce
+   ```
+
+   For more information on installing Docker please refer to the
+   [Docker Guide](https://docs.docker.com/engine/installation/linux/centos).
+
+2. Configure Docker to use Kata Containers by default with the following commands:
+
+   ```bash
+   $ sudo mkdir -p /etc/systemd/system/docker.service.d/
+   $ cat <<EOF | sudo tee /etc/systemd/system/docker.service.d/kata-containers.conf
+   [Service]
+   ExecStart=
+   ExecStart=/usr/bin/dockerd -D --add-runtime kata-runtime=/usr/bin/kata-runtime --default-runtime=kata-runtime
+   EOF
+   ```
+
+3. Restart the Docker systemd service with the following commands:
+
+   ```bash
+   $ sudo systemctl daemon-reload
+   $ sudo systemctl restart docker
+   ```
+
+4. Run Kata Containers
+
+   You are now ready to run Kata Containers:
+
+   ```
+   $ sudo docker run -ti busybox sh
```

Comment:
just a thought - I know we have this as a non-bash backtick as we cannot automatically test the `sh` invocation, as it would hang... how about we switch it to:

```bash
$ sudo docker run -ti busybox uname -a
```
(or the `bash -c 'uname -a'` if that is what is needed) - then we can both run that in the automation, and have it print out the in-VM kernel info to show we are in a VM and not on the host.

Reply:
like it - fixed.